contents/javascript/Javascript-framework-VueJs.md

```diff
@@ -13,9 +13,14 @@
 
 # VueJs
 
-Author: [Lu Lechuan](https://github.com/LuLechuan)
+**Authors**: [Lu Lechuan](https://github.com/LuLechuan), [Chelsey Ong](https://github.com/chelseyong)<br>
+Reviewers: [Ong Shu Peng](https://github.com/ongspxm), [Gilbert Emerson](https://github.com/emer7)
 
-VueJs (also known as Vue) is an open-source [JavaScript framework](https://en.wikipedia.org/wiki/JavaScript_framework) for building user interfaces. It is designed to improve code quality and maintainablity.
+VueJs (also known as Vue) is an open-source [JavaScript framework](https://en.wikipedia.org/wiki/JavaScript_framework) for building user interfaces. It is designed to improve code quality and maintainability.
+
+<box type="info">
+Reader should have basic knowledge of HTML and JavaScript before proceeding with this article.
```

Comment:
-> `This article assumes the reader has a basic knowledge of HTML and JavaScript.`